### PR TITLE
fix(environments): Prevent user reports fetching twice when env changes

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/projectUserReports.jsx [hunk 1]
```diff
@@ -52,7 +53,11 @@ const ProjectUserReports = createReactClass({
   },
 
   componentWillReceiveProps(nextProps) {
-    if (nextProps.location.search !== this.props.location.search) {
+    // Ignore changes to environment term since this is handled separately
+    const nextSearchTerm = omit(qs.parse(nextProps.location.search), 'environment');
+    const thisSearchTerm = omit(qs.parse(this.props.location.search), 'environment');
+
+    if (!isEqual(nextSearchTerm, thisSearchTerm)) {
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/projectUserReports.jsx]: if they both change, this will still fetch twice, right?

Comment (MaxBittker) [reply src/sentry/static/sentry/app/views/projectUserReports.jsx]: (its still an improvement so no worries)

**Review by MaxBittker — APPROVED** [review]
fix looks good 👍

File: src/sentry/static/sentry/app/views/projectUserReports.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/projectUserReports.jsx]: I guess so, but they wouldn't both change at the same time?

**Pull request merged**